suites/google/googleServiceAccountKeyTest.js

```diff
@@ -251,7 +269,14 @@ Scenario('SA key removal job test: remove expired creds @reqGoogle', async (fenc
   await fence.do.deleteTempGoogleCreds(
     creds0Key,
     users.user0.accessTokenHeader,
-  );
+  ).then((deletionResult) => {
+    console.log(`deletionResult: ${JSON.stringify(deletionResult)}`);
+    console.log(`is it an error?: ${deletionResult instanceof Error}`);
+    chai.expect(
```

Comment:
if this fails here, the `files.deleteFile(` below won't run. this is probably fine but just a note that the first failing assert ends the scenario

Reply:
We clean-up workspaces on every test run so the Jenkins file system for that PR run is wiped out.
That is, however, a real issue for whoever is running this locally (But, in case that happens, they would see some weird file in their `git status` output so... there is a manual workaround for this edge case).